### ISSUE TITLE
Animated.event support on onScroll

### ISSUE THREAD

**Issue by naqvitalha** [Shopify/flash-list#81]
# What
Developers can pass an Animated.event to FlatList and ScrollView to sync some animations with scroll offset. If used with native drive this can work on native side completely. This is missing from RFL today.

More info:
While RFL and RLV use ScrollVIew internally they don't expose scrollableNode. If that can be added or we pass our own scrollview to RLV we can add this function. `createAnimatedComponent` can work on any component that can implement getScrollableNode function and return ref to the scrollview.

So, in summary we need to implement getScrollableNode method inside RFL which should return ref of internal ScrollView. Once this is done we can just call `createAnimatedComponent(RFL)` and export this as AnimatedRFL. In theory, this should work.
